scripts/collect_steps.py

```diff
@@ -19,7 +19,7 @@
     "--num-multi-bits",
     "3",
 ]
-QUERY_SIZE = 10
+QUERY_SIZE = 100
```

Comment:
Changed this here to avoid issues in future

Reply:
would it be sufficient to run malicious IPA for 1M rows? It seems that number of steps synthesized depends on the depth of the circuit - in this case would we be better generating all of them, similarly to what we did before?